ssh/docker-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# file made while testing
+echo "Starting sshd service.."
+cat /tmp/sshtest/.ssh/sshrucio.key.pub >> /root/.ssh/authorized_keys
+cat /root/.ssh/authorized_keys
+cat /etc/ssh/sshd_config
+/usr/local/bin/dumb-init /usr/sbin/sshd -D
+
```

Comment:
@bziemons is this right? because it seems to not work even then. I tried as you had said in the container before the entrypoint and that had not worked either :/

Reply:
no, you are missing the double dash: `--` and you should also run this script with dumb-init, so use

```Dockerfile
ENTRYPOINT ["/usr/local/bin/dumb-init", "--"]
CMD ["/bin/bash", "/docker-entrypoint.sh"]
```

if I guessed correctly what you want to do.

Reply:
@bziemons it still does not work, im not too sure why

Reply:
could you please push the current state so I can have another look?